src/open_samus_returns_rando/files/templates/randomizerpowerup.lua

```diff
@@ -51,7 +51,7 @@ function RandomizerPowerup.OnPickedUp(resources, actorOrName)
 
     Scenario.UpdateProgressiveItemModels()
     if actorOrName ~= nil then
-        RandomizerPowerup.MarkLocationCollected(actorOrName)
+        RandomizerPowerup.GetActorName(actorOrName)
```

Comment:
Function name is super misleading. `Get` but it doesn't return an actor and also not a name. In fact, it doesn't return anything.

Reply:
What do you suggest then?

Reply:
I'm bad with names :D 
The main thing it does is actually calling the mark location and special energy function. Usually you name something after the stuff it does and not after what parameters it takes.